src/arrayReverse.test.js

```diff
@@ -1,20 +1,47 @@
-'use strict';
+"use strict";
 
 describe(`Function 'arrayReverse':`, () => {
-  const arrayReverse = require('./arrayReverse');
+  const arrayReverse = require("./arrayReverse");
 
   it(`should be declared`, () => {
     expect(arrayReverse).toBeInstanceOf(Function);
   });
 
-  it(`should return an array`, () => {
+  it(`should handle single-word arrays`, () => {
+    expect(arrayReverse(["Hello"])).toEqual(["olleH"]);
+  });
+
+  it(`should reverse the strings and their order in the array`, () => {
+    expect(arrayReverse(["Mate", "Academy"])).toEqual(["ymed", "acAetaM"]);
```

Comment:
The expected output should reverse both the strings and their order in the array. The correct expected output should be `["ymedacA", "etaM"]`.